server/src/main/java/org/candlepin/guice/CandlepinModule.java

```diff
@@ -148,6 +137,24 @@
 import com.google.inject.name.Names;
 import com.google.inject.persist.jpa.JpaPersistModule;
 
+import org.hibernate.cfg.beanvalidation.BeanValidationEventListener;
+import org.hibernate.validator.HibernateValidator;
+import org.hibernate.validator.HibernateValidatorConfiguration;
+import org.quartz.JobListener;
+import org.quartz.spi.JobFactory;
+import org.xnap.commons.i18n.I18n;
+
+import java.util.Properties;
```

Comment:
What is the correct order? From the candlepin.importorder it seems that Eclipse is ordering the imports correctly.

org.candlepin and org.hibernate shouldn't be next to each other and com.* should be between them

candlepin.importorder:
{{{
7=javax
6=java
5=ch
4=net
3=org
2=com
1=com.redhat.rhn
0=org.candlepin
}}}